### ISSUE TITLE
Make file details available with every file type.

### ISSUE THREAD

**Issue by RohanRK0** [shibayan/WinQuickLook#38]
Sometime we need to see the file details such as size or last modified time which is what we see with folders, shortcut or any unsupported file types but with the case of supported files such as pdf, docx, txt, image and videos we don't get the option to see file description instead of the file itself. So in my opinion there should be an option to view the generic preview (used for non supported file types) of the supported files as well.

That can be implemented by replacing the open with associated program button at the top left with file information button, since we can simply open any file by double clicking or pressing enter on the file. Open with associated program action can be assigned to double clicking in the preview box.

Speaking of the Generic Preview, there can be "Created" detail along with 'size' and 'last modified' if the developer deems it useful. :)

**Comment by shibayan** [comment]
I think it's a good idea. I'd like to consider a pop-up that can display the metadata that the file has

**Comment by RohanRK0** [comment]
> I think it's a good idea. I'd like to consider a pop-up that can display the metadata that the file has

Yes that would be even better, in that case the file information button should appear in unsupported file types or in generic preview as well, which will show more detailed info than the generic preview, such as date created, date accessed, location etc.
